app/src/main/kotlin/com/wire/android/WireApplication.kt

```diff
@@ -116,5 +135,21 @@ class WireApplication : Application(), Configuration.Provider {
 
     private companion object {
         const val LONG_TASK_THRESH_HOLD_MS = 1000L
+
+        @Suppress("MagicNumber")
+        enum class MemoryLevel(val level: Int) {
+            TRIM_MEMORY_BACKGROUND(40),
+            TRIM_MEMORY_COMPLETE(80),
+            TRIM_MEMORY_MODERATE(60),
+            TRIM_MEMORY_RUNNING_CRITICAL(15),
+            TRIM_MEMORY_RUNNING_LOW(10),
+            TRIM_MEMORY_RUNNING_MODERATE(5),
+            TRIM_MEMORY_UI_HIDDEN(20),
+            TRIM_MEMORY_UNKNOWN(-1);
```

Comment:
Can't we instead of using the hard-coded numbers, rely on the constants defined in the SDK?
Less maintenance 😅 

```suggestion
            TRIM_MEMORY_BACKGROUND(ComponentCallbacks2.TRIM_MEMORY_BACKGROUND),
            TRIM_MEMORY_COMPLETE(ComponentCallbacks2.TRIM_MEMORY_COMPLETE),
            TRIM_MEMORY_MODERATE(ComponentCallbacks2.TRIM_MEMORY_MODERATE),
            TRIM_MEMORY_RUNNING_CRITICAL(ComponentCallbacks2.TRIM_MEMORY_RUNNING_CRITICAL),
            TRIM_MEMORY_RUNNING_LOW(ComponentCallbacks2.TRIM_MEMORY_RUNNING_LOW),
            TRIM_MEMORY_RUNNING_MODERATE(ComponentCallbacks2.TRIM_MEMORY_RUNNING_MODERATE),
            TRIM_MEMORY_UI_HIDDEN(ComponentCallbacks2.TRIM_MEMORY_UI_HIDDEN),
            TRIM_MEMORY_UNKNOWN(-1);
```

Reply:
Good catch ! done